Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -107,44 +119,43 @@ for module in "${HYPERV_MODULES[@]}"; do
 
     # Check to see if the module is loaded
     if [[ $load_status =~ $module ]]; then
-        if [ "$rpmAvailable" = true ] ; then
-            if [ ! -z "$isLISInstalled" ]; then
-                version=$(modinfo "$module" | grep version: | head -1 | awk '{print $2}')
-                LogMsg "$module module: ${version}"
-                if [ "$module" == "mlx4_en" ] ;then
-                    if [ "$MLNX_VERSION" != "$version" ] ;then
-                        LogErr "ERROR: Status: $module $version doesnot match with build version $MLNX_VERSION"
-                        exit_code="1"
-                    fi
-                    continue
-                fi
-                if [ "$expected_lis_version" != "$version" ] ;then
-                    LogErr "ERROR: Status: $module $version doesnot match with build version $expected_lis_version"
-                    exit_code="1"
-                fi
+        LogMsg "LIS module $module loaded successfully"
+        if [ "$rpmAvailable" = true ] && [ ! -z "$isLISInstalled" ]; then
+            version=$(modinfo "$module" | grep version: | head -1 | awk '{print $2}')
+            LogMsg "$module module version: ${version}"
+            if [ "$module" == "mlx4_en" && "$MLNX_VERSION" != "$version" ] ;then
+                    LogErr "Status: $module $version did not match to the build one, $MLNX_VERSION"
```

Comment:
indent

Reply:
Fixed in 69aa5c4f471226535ca1a0bcb9683c41efe546b4

Reply:
indent issue still there for line 127 and 128

Reply:
fixed in 78f5c4a94ea94a2156d5d01eebe191584c64fcdf